README.md

```diff
@@ -1,145 +1,132 @@
-
-```go
-package main
-func main() {}
-```
-
-Finally, from the root of the new module, I ran `vgo get arschles.com/testmodule@v1.0.0` and got the
-following output:
-
-```console
-$ vgo get arschles.com/testmodule@v1.0.0
-vgo: downloading arschles.com/testmodule v1.0.0
-vgo: import "arschles.com/testmodule": zip for arschles.com/testmodule@v1.0. has unexpected file testmodule/.DS_Store
-```
-
-As you can see, the CLI uploaded a file to athens that's not `.go`, `go.mod`, or anything else
-that vgo, so at least the CLI needs some work (and the server needs some sanity checks too).
-
-You can get around all of this by manually zipping up your code and uploading it with `curl` or
-similar, but like I said, that's super impractical. Yay alpha software!
+# Athens
+
+_This is a very early alpha release, and the API will be changing as the proxy API changes._
+_Do not run this in production. This warning will be changed or removed as the project and the proxy API changes._
+
+## What does it do?
+
+1) Proxy server for the modules defined in vgo
+2) Package registry (like npm and ruby gems) for modules
+
+Athens is a proxy server for [vgo modules](https://github.com/golang/go/wiki/vgo). It implements the download protocol specified [here](https://research.swtch.com/vgo-module) (under "Download Protocol"), and a few additional API endpoints to make it more useful. See [API.md](./API.md) for more information.
+
+# Architecture
+
+Athens is composed roughly of three logical pieces. The below list contains links
+to a description of each:
+
+* [Module proxy](./PROXY.md)
+* [Module registry](./REGISTRY.md)
+* [CLI](./CLI.md)
+
+# Development
+
+The server is written using [Buffalo](https://gobuffalo.io/), so it's fairly straightforward to get started on development. You'll need Buffalo v0.11.0 or later to do devlopment on Athens.
+
+Download
+[v0.11.0](https://github.com/gobuffalo/buffalo/releases/tag/v0.11.0) or later, untar/unzip the binary into your PATH, and then run the following from the root of this repository:
+
+```console
+buffalo dev
+```
+
+You'll see some output in your console that looks like this:
+
+```console
+$ buffalo dev
+buffalo: 2018/02/25 16:09:36 === Rebuild on: :start: ===
+buffalo: 2018/02/25 16:09:36 === Running: go build -v -i -o tmp/vgoprox-build  (PID: 94067) ===
+buffalo: 2018/02/25 16:09:37 === Building Completed (PID: 94067) (Time: 1.115613079s) ===
+buffalo: 2018/02/25 16:09:37 === Running: tmp/vgoprox-build (PID: 94078) ===
+time="2018-02-25T16:09:37-08:00" level=info msg="Starting application at 127.0.0.1:3000"
+INFO[2018-02-25T16:09:37-08:00] Starting Simple Background Worker
+
+Webpack is watching the files…
+```
+
+After the `Starting application at 127.0.0.1:3000` is logged, the server is up and running.
+As you edit and save code, Buffalo will automatically restart the server. **This means that all of your modules will disappear** because the only storage driver is in-memory right now.
```

Comment:
we have disk storage now as well

Reply:
could change wording to:

```
By default, athens stores modules in memory, so all of your modules will disappear on restart.
```

... or similar

note for a later PR: we may want to default to disk storage in a follow-up PR